### DISCUSSION TITLE
Blade component for multiple selectbox

### DISCUSSION THREAD

**Discussion by ayemin-tun** [filamentphp/filament#9503] [Ideas]
when I using blade component for select I found only for single select. Please Can You make for multiselect that similarly in filament resource.

**Comment by sinnbeck** [comment]
Is this what you are looking for ? https://filamentphp.com/docs/3.x/forms/fields/select#multi-select

If you mean this https://filamentphp.com/docs/3.x/support/blade-components/select just add `multiple` to the `x-filament::input.select`

```
 <x-filament::input.wrapper>
    <x-filament::input.select wire:model="status" multiple>
        <option value="draft">Draft</option>
        <option value="reviewing">Reviewing</option>
        <option value="published">Published</option>
    </x-filament::input.select>
</x-filament::input.wrapper>

**Reply by ayemin-tun** [reply]
![Screenshot (193)](https://github.com/filamentphp/filament/assets/76741291/2853cb06-9423-4fd8-9a07-bc8cb4d4f63a)
It use this but result design is this not like this design
![Screenshot (194)](https://github.com/filamentphp/filament/assets/76741291/74982c19-659f-4132-9d4b-cf65653e151f)

**Reply by sinnbeck** [reply]
Correct. As the documentation states

>The select component is a wrapper around the native `<select>`

So not the Choices-js version :)
That version needs a lot of set up from livewire to work, so it wouldnt work if you just add the component code

I suggest you use a form https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component

**Reply by ayemin-tun** [reply]
But my project need this to create like blade component so I use select 2 lib but the design is not match with my filament theme. I want to change this to match with filament theme.

**Reply by sinnbeck** [reply]
If you want to use select2 just add a plain `<select>` and add some javascript to initialize it and such? Filament does not use select2

**Comment by marcosmarcolin** [comment]
@ayemin-tun Hi, I am facing the same situation, have you found or implemented any solution for this?

I have forms with dynamic fields and I wouldn't like to deal with this manually, the multiple option would be of great value.

**Comment by ihahachi** [comment]
Same situation... no solution for me.

**Reply by marcosmarcolin** [reply]
@ihahachi 

I solved it by creating a [livewire component](https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component) and rendering it on the Filament page.

```blade
<x-filament-panels::page xmlns:x-filament="http://www.w3.org/1999/html" xmlns:x-fillament="http://www.w3.org/1999/html">
    @livewire('my_component', ['param_1' => 'value_1'])
</x-filament-panels::page>
```

So I built the entire form in PHP with `FormBuilder`, it was much easier, I should have done this from the beginning, I wasted a lot of time trying with Blade. ;(
